src/sidebar/CustomModelBox.tsx

```diff
@@ -0,0 +1,84 @@
+import 'codemirror/lib/codemirror.css'
+import 'codemirror/addon/hint/show-hint.css'
+import 'codemirror/addon/lint/lint.css'
+// todonow: this belongs to this app and we should not take it from the demo...
+import 'custom-model-editor/demo/style.css'
```

Comment:
I was not sure how to add (where to put the file) global css rules as we need them for codemirror like `.cm-s-default`.